styles/OurClients.module.css

```diff
@@ -0,0 +1,30 @@
+.our_clients {
+  margin-top: 64px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 0px;
+  border-bottom: 1px solid black;
+  padding-bottom: 64px;
+}
+
+.our_clients__text {
+  font-family: "Roboto Mono";
+  font-style: normal;
+  font-weight: 400;
```

Comment:
I don't think we use a font like this way. Do a global search of the key word "fontRobotoMono" to see how our group uses the font in components.